wiki/markdown/usage.md

```diff
@@ -0,0 +1,6 @@
+## Usage
+To get information about configuration and usage of REMReM Publish CLI see [here](https://eiffel-community.github.io/eiffel-remrem-publish/cliUsage.html).
```

Comment:
To utilize GitHub's version handling of the documentation (in case of changes) it's better to use relative links to point directly to the markdown files in the repository.